.drone.star

```diff
@@ -2160,33 +2162,40 @@ def idpService():
         }],
     }]
 
-def ocisService():
+def ocisService(type):
```

Comment:
If we do this here we can revome the line change in `1341`
```suggestion
def ocisService( type = "acceptance-tests"):
```

Reply:
I just want to keep uniformity for using type.